### DISCUSSION TITLE
Add documentation about unit testing Next.js apps

### DISCUSSION THREAD

**Discussion by Mikilll94** [vercel/next.js#23801] [Ideas]
### Describe the feature you'd like to request

It would be nice to add some documentation about unit testing Next.js apps. Right now docs contain completely no information about unit testing.

Personally, I am curious what is the recommended way of unit testing the Next.js router. I saw this long discussion: https://github.com/vercel/next.js/issues/7479 but it seems that there is no conclusion what are the best ways to test the Next.js router.

### Describe the solution you'd like

Add info to the docs about unit testing Next.js apps. It is important to mention the ways of testing the Next.js router.

### Describe alternatives you've considered

An alternative solution is to add an example project which showcases how to test the Next.js router.

**Comment by stefanprobst** [comment]
related: https://github.com/vercel/next.js/issues/10967
